app/uk/gov/hmrc/economiccrimelevyregistration/models/Registration.scala

```diff
@@ -33,4 +34,15 @@ final case class Registration(
 
 object Registration {
   implicit val format: OFormat[Registration] = Json.format[Registration]
+
+  def empty(internalId: String): Registration = Registration(
+    internalId = internalId,
+    entityType = None,
+    meetsRevenueThreshold = None,
+    amlSupervisor = None,
+    incorporatedEntityJourneyData = None,
+    soleTraderEntityJourneyData = None,
+    partnershipEntityJourneyData = None,
+    lastUpdated = Some(Instant.ofEpochSecond(1))
+  )
```

Comment:
not sure we need this, as we only create an empty one in the `RegistrationRepositorySpec`, so it feels like a smell to have a test helper method in the app code?

Reply:
Redundant method removed.